### ISSUE TITLE
Column 'dataset_name' not found when reading sys.reflections

### ISSUE THREAD

**Issue by peter-wimsey** [fabrice-etanchaud/dbt-dremio#30]
Hi there,

I am using dbt-dremio 1.0.6.5 and Dremio 19.4.0 and I am getting `Column 'dataset_name' not found in any table` on Dremio.

Not sure I understood #22 correctly, but should this combination of versions be working?

Running the query from #23

```sql
select 
    case when count(column_name) = 1
        then 'dataset'
        else 'dataset_name'
    end 
from information_schema.columns
where table_schema = 'sys'
    and table_name   = 'reflections'
    and column_name  = 'dataset'
```
returns `dataset` as expected.

**Comment by fabrice-etanchaud** [comment]
Hi Peter,
thank you for giving dbt-dremio a try !
I am afraid the last version will only work on dremio 20 and later.

A quick workaround would be to disable reflection handling (see readme).
If you create reflections, let me know, I will try to add a configuration variable.
Be aware that incremental materialization will only work with a iceberg enabled dremio platform.

Looking forward,
Fabrice

**Comment by peter-wimsey** [comment]
Hi Fabrice,

Thanks for your message and apologies for my late reply.

For some context, I am moving an existing workflow to a new environment.
I have only skimmed the SQL, not even sure if we create reflections or not. 

For the time being I downgraded dbt-dremio, our requirements.txt:
```
dbt-core==0.21.0
dbt-dremio
markupsafe<2.1.0  # see https://github.com/pallets/markupsafe/issues/282
```
which seems to work fine. I will upgrade dremio-dbt once we upgrade Dremio.

Thanks for maintaining this library!

Kind regards, 
Peter
